drivers/adcxx1c/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = adcxx1c
```

Comment:
just for future reference: no need for this line, if you dont define `MODULE`, it will take the folder name per default...